package.json

```diff
@@ -11,15 +11,13 @@
     "dist/**/*"
   ],
   "scripts": {
-    "build-docs": "node make-webgpu-docs.mjs",
-    "build-ci": "npm run build-docs",
+    "test": "for t in tests/* ; do (cd \"$t\" && npm i && npm test); done",
```

Comment:
will this run on windows? Does it matter? A small node script would be portable but if it doesn't matter than never mind.

Reply:
Pretty sure it won't work on Windows. I thought about it, but didn't feel like it was necessary to handle right now, since I mainly wanted it to work in the github action. Maybe if someone actually runs into it later on, they'll feel inspired to fix it :P